en_US/security/authn/jwt.md

```diff
@@ -4,181 +4,73 @@
 
 ## Authentication Principle
 
-A client sends JWT as a password. When accepting the connection, EMQX Broker uses the configured key or secret to verify
-JWT signature. If the signature is not successfully verified, the authentication fails. On success, the JWT authenticator
-verifies JWT _claims_ (payload "fields") like `exp` (expiration time), etc. for validity. A user may
-provide some additional checks. If all the checks succeed, the authentication succeeds too. Otherwise, the authentication fails.
+The client carries the JWT in the connection request, and EMQX uses the pre-configured secret or public key to verify the JWT signature. If the user is using JWKS, the JWT authenticator attempts to verify the JWT signature using all currently valid public keys. If the signature verification is successful, the JWT authenticator proceeds to check the claims. If there are claims such as `iat`, `nbf` or `exp`, the JWT authenticator will actively check the validity of the JWT based on these claims. In addition to this, we also allow users to specify some additional claims checks. The client is finally allowed to login only if the signature verification and claims check pass together.
 
-Sample JWT:
-```
-eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2NTQyNTQ2MDEsInVzZXJuYW1lIjoibXl1c2VyIn0.yi1FSkrLbd9qv4-TrfEa8opiQ9NqShgvMT2lG3pzhgA
-```
+## Common Usage
 
-The decoded header is:
-```json
-{
-  "alg": "HS256",
-  "typ": "JWT"
-}
-```
+The name of the JWT authenticator may be misleading, but in essence it just checks the signature of the JWT, which means that the JWT authenticator does not guarantee the legitimacy of the client's identity.
```

Comment:
```suggestion
Used in the most basic way, the JWT authenticator just checks the signature of the JWT, which means that the JWT authenticator does not guarantee the legitimacy of the client's identity.
```

Reply:
I mean, EMQX actually _can_ verify identity, if STS provides appropriate tokens.

Reply:
But when the client gets the JWT, it usually means that it has been authenticated once. At most, EMQX checks whether the JWT matches the current holder.